electroncash/tests/regtest/util.py

```diff
@@ -54,15 +59,25 @@ def poll_for_answer(url: Any, json_req: Any = None, expected_answer: Any = None,
             pass
         time.sleep(poll_interval)
         current = time.time()
+    raise TimeoutError("Timed out waiting for an answer")
 
-def bitcoind_rpc_connection() -> AuthServiceProxy:
+def get_bitcoind_rpc_connection() -> AuthServiceProxy:
+    global _bitcoind
```

Comment:
Nit: technically `global` only needed when writing to a variable.. When I see this I think "oh this function modifies `_bitcoind`.. let me scroll down and see where...".  And.. here it doesn't modify it :'(.

Not a huge deal. Can leave this as-is.. just wanted to point it out is all.

Reply:
I'll change it

Reply:
Ok, done. Thanks.